x-pack/plugins/cases/public/components/app/index.tsx

```diff
@@ -44,7 +44,7 @@ const CasesAppComponent: React.FC<CasesAppProps> = ({
         permissions: userCapabilities.generalCases,
         basePath: '/',
         features: { alerts: { enabled: false } },
-        releasePhase: 'experimental',
+        releasePhase: 'ga',
```

Comment:
consider getting rid off this line as the default seems to be `ga`

Tested locally and is working as expected